addons/website_sale/tests/test_website_sale_gmc.py

```diff
@@ -0,0 +1,701 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import io
+import time
+from datetime import datetime, timedelta
+from lxml import etree
+from PIL import Image
+from werkzeug.exceptions import NotFound
+
+from odoo import Command
+from odoo.tests import HttpCase, tagged
+
+from odoo.addons.website.tools import MockRequest
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleGMCCommon(HttpCase):
```

Comment:
Can't you rely on existing commons to simplify your setup ?
Maybe the `ProductAttributesCommon` ?